js/main.js

```diff
@@ -29,6 +29,13 @@ var TYPE_OF_HOUSES = [
   'bungalo'
 ];
 
+var housesNames = {
```

Comment:
```suggestion
var HousesNames = {
```
https://up.htmlacademy.ru/javascript/19/criteries#b8
<img width="1230" alt="Screenshot 2020-02-11 at 23 16 02" src="https://user-images.githubusercontent.com/6417966/74275221-8c866200-4d24-11ea-89f6-8a3864e9509b.png">